spritzle/resource/torrent.py

```diff
@@ -186,6 +186,8 @@ def generate_torrent_info(data):
 
     for tid in tids:
         handle = get_valid_handle(core, tid)
+        await core.resume_data.delete(handle.name())
```

Comment:
I believe that session.remove_torrent() is not exactly synchronous so we should be doing this clean-up in the torrent_removed_alert.

Reply:
But, do we really care here? User has requested it be deleted, why does it matter if we wait for libtorrent alert to get rid of resume data?

Reply:
Potential race condition because the torrent is still in the session until the alert fires so we should keep all state on the torrent until its actually removed.

Reply:
Taken care of now.